prime-dotnet-webapi/ViewModels/Emails/SiteNotificationEmailViewModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace Prime.ViewModels.Emails
+{
+    public class SiteNotificationEmailViewModel
```

Comment:
I would rather this ViewModel be specific to the one email using it rather than having a more generic name. How about `SiteReviewedEmailViewModel`.